include/sdf/Element.hh

```diff
@@ -443,6 +451,9 @@ namespace sdf
     /// \brief True if element is required
     public: std::string required;
 
+    /// \brief True if the element is set.
+    public: bool set;
```

Comment:
nit: Can we use a more descriptive variable? eg. `isExplicitlySet` or `isSetByUser`